experimental/material_3_demo/lib/component_screen.dart

```diff
@@ -2182,6 +2182,101 @@ class _SlidersState extends State<Sliders> {
   }
 }
 
+class SearchAnchors extends StatefulWidget {
+  const SearchAnchors({super.key});
+
+  @override
+  State<SearchAnchors> createState() => _SearchAnchorsState();
+}
+
+class _SearchAnchorsState extends State<SearchAnchors> {
+  String? selectedColor;
+  List<ColorItem> searchHistory = <ColorItem>[];
+
+  Iterable<Widget> getHistoryList(SearchController controller) {
+    return searchHistory.map((color) => ListTile(
```

Comment:
Two space indent though out this method and the next one

Reply:
Yeah, the format is odd here but it is caused by the `dart format` command:(

Reply:
OK, I see.